app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -74,6 +73,29 @@ class ScrollingActivity : AppCompatActivity() {
         recyclerView.addOnScrollListener(object : RecyclerView.OnScrollListener() { })
     }
 
+    private fun initButtons(){
+        initAddUserItemButton()
+    }
+
+    private fun initAddUserItemButton(){
+        addUserItemFab= findViewById(R.id.add_user_item_fab)
+        addUserItemFab.setOnClickListener {
+            val createNewItemIntent = Intent(applicationContext, NewUserItem::class.java)
+            startActivityForResult(createNewItemIntent,NEW_USER_ITEM_RESULT_CODE)
+        }
+    }
+
+    @Override
```

Comment:
@enavhasson in my opinion this should be done in NewUserItem.kt class, using an 'OnClick' method from a 'approve' button. I don't think scrolling activity should deal with it